include/hse/types.h

```diff
@@ -55,6 +55,7 @@ typedef uint64_t hse_err_t;
 /** @brief Error context values */
 enum hse_err_ctx {
     HSE_ERR_CTX_NONE, /**< No context */
+    HSE_ERR_CTX_TXN_EXPIRED, /**< Transaction timed out */
```

Comment:
Should we add the write conflict code while we're here?

Reply:
I would rather do it in another PR, but I have the time to do it if you want me to.